components/script/dom/webidls/Element.webidl

```diff
@@ -127,3 +127,4 @@ Element includes ChildNode;
 Element includes NonDocumentTypeChildNode;
 Element includes ParentNode;
 Element includes ActivatableElement;
+Element includes ARIAMixin;
```

Comment:
The [ARIAMixin spec](https://w3c.github.io/aria/#dom-ariamixin) has this `includes` statement there, but we have all our `Element includes` here. I wasn't sure which should take precedence.